quadrat/inc/block-patterns.php

```diff
@@ -43,3 +43,59 @@ function quadrat_register_block_patterns() {
 endif;
 
 add_action( 'init', 'quadrat_register_block_patterns', 9 );
+
+if ( ! function_exists( 'quadrat_register_illustration_patterns' ) ) :
+
+	function quadrat_register_illustration_patterns() {
+		if ( function_exists( 'register_block_pattern_category' ) ) {
+			register_block_pattern_category(
+				'quadrat-images',
+				array( 'label' => __( 'Quadrat Images', 'quadrat' ) )
+			);
+		}
+
+		if ( function_exists( 'register_block_pattern' ) ) {
+			$illustration_patterns = array(
+				'circle-rolling-over-a-triangle' => __( 'Circle rolling over a triange', 'quadrat' ),
+				'combination-of-purple-triangles' => __( 'Combination of purple triangles' , 'quadrat' ),
+				'combination-of-white-orange-and-pink-triangles' => __( 'Combination of white orange and pink triangles' , 'quadrat' ),
+				'complex-shape-combination-and-two-pink-squares-on-a-dark-purple-background' => __( 'Complex shape combination and two pink squares on a dark purple background' , 'quadrat' ),
+				'group-of-squares-displayed-diagonally' => __( 'Group of squares displayed diagonally' , 'quadrat' ),
+				'group-of-white-pink-and-orange-circles' => __( 'Group of white pink and orange circles' , 'quadrat' ),
+				'many-pink-squares-overlapped' => __( 'Many pink squares overlapped' , 'quadrat' ),
+				'orange-and-pink-shapes-displayed-in-different-ways' => __( 'Orange and pink shapes displayed in different ways' , 'quadrat' ),
+				'orange-triangle-in-purple-background' => __( 'Orange triangle in purple background' , 'quadrat' ),
+				'pink-square-and-dark-purple-triangle' => __( 'Pink square and dark purple triangle' , 'quadrat' ),
+				'pink-square-with-dark-shadow-on-orange-background' => __( 'Pink square with dark shadow on orange background' , 'quadrat' ),
```

Comment:
The file for this one starts with an uppercase P, we may want to fix that

Reply:
fixed